C_CPP/Algorithmic Approaches/Brute Force/roman_to_integer.md

```diff
@@ -0,0 +1,117 @@
+# Roman To Integer Using Brute Force Approach
+
+## Brute Force
+A Brute Force Algorithm is a straight-forward approach to a problem i.e the first method or algorithm which come in our mind to solve a particular problem. In this we try every possibility rather than going for advance techniques to improve efficiency.
+
+## Roman To Integer
+
+Task: Given a roman numeral, convert it to an integer.
+
+Approach: Roman numerals are represented by seven symbols: I, V, X, L, C, D and M.
+
+![image](https://user-images.githubusercontent.com/73405757/140963448-360c6a7f-f391-4396-8fbd-68ab3c365c52.png)
+
+[Reference source to image](https://leetcode.com/problems/roman-to-integer/)
```

Comment:
Change this to:
"Source: [LeetCode](https://leetcode.com/problems/roman-to-integer/)"

Reply:
Changed, please check